src/main/java/com/onaio/steps/activities/SettingsActivity.java

```diff
@@ -93,4 +89,15 @@ public void enableParticipantFlow(View view) {
         prepareViewWithData();
     }
 
+    public void eraseData(View view) {
+        DialogInterface.OnClickListener confirmListener = new DialogInterface.OnClickListener() {
+            @Override
+            public void onClick(DialogInterface dialog, int which) {
+                db.truncate();
```

Comment:
Do we have a confirmation for the deletion? This sounds like where someone may need to confirm before the deletion is done. Or is it not required for this?